### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by pacyL2K19**
# The aim of this PR 

>In this project I was designing the old version of Apple Home page
## What I did : 
- I designed the header
- I designed the main content of the page 
- I put the footer 
## What it remains to do 
- Work on your differents feedbacks

### CODE REVIEW

**Review by Temmarie — CHANGES_REQUESTED** [review]
### Changes Required! :recycle:
Hi @pacyL2K19 ! 👋 👋 👋 
### You have done well so far :clap: However, your project cannot be approved until you make some changes. Kindly work on them and submit them for another code review.


**General** 

- [x] Your readme file has some errors in it, please update it.

**Recommended**
- **Navigation bar**
- [x] The navigation bar has a gradient that while it's nice doesn't look quite as close as the actual site. It doesn't have to be the exact gradient but bring it as close as you possibly can. 

- [x] You used a font family that's different from that on the actual page. It should be ` font-family:  "Lucida Grande", "Lucida Sans Unicode", Helvetica, Arial, Verdana, sans-serif`

You can also use SVG images instead of using text for the navbar, use the [SVG Export chrome extension](https://chrome.google.com/webstore/detail/svg-export/naeaaedieihlkmdajjefioajbbdbdjgp?hl=en-GB) to get the images. 

_The above suggestion on the use of SVGs also apply to the main text of the page, although it is optional, as it looks quite okay right now._ 



- **Footer**

- [x] The footer links should be smaller in size and should use the font-family suggested above.

 
__You've done an exceptional job so far! 
Kindly make the necessary changes and submit another code review request.__
_Feel free to reach me on slack for any inquiries about this review._ :wink:
__Crace Tamara Ekunola __
[GitHub](https://github.com/temmarie) | [LinkedIn](https://www.linkedin.com/in/ekunola-grace/) | [Twitter](https://twitter.com/TemmarieW)

File: README.md [hunk 1]
```diff
@@ -1,2 +1,48 @@
-# old-apple-page
-A clone of the old version of apple home page
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Old Apple page 
+
+> This project is a clone of the Apple old home page
+
+<h4>Screeshots</h4>
```
Comment (arslanbisharat) [inline README.md:5]: [Optional] You can remove this text

**Review by arslanbisharat — APPROVED** [review]
# Approved :smile: 

Hello Developers :wave:

I just reviewed your project and I am very happy to announce that your project is approved.
Congratulations on achieving this milestone in your career. :clap:

This project is approved! Congratulations!. There are still some minor changes that you can make to improve your project but they are optional. You can implement them at any point in time.

- [ ] [Optional] You can add **styles** and **assets** folder in _a single folder_. This will help you to make your project clean.  

- [x] [Optioanl] You can add more information in the About section of your repo. About section describes what you did in the project. 

- [ ] 

_I am looking forward to seeing your next project._

-----------------------------------------------------------------------------------------------

**Regards,
Arslan Bisharat
slack: arslanbisharat**

File: README.md (hunk 1 above)
Comment (pacyL2K19) [reply README.md:5]: Thank you for your feedback dear

Comment (arslanbisharat) [reply README.md:5]: You welcome 😄

**Pull request merged**